test-harness/template.yaml

```diff
@@ -43,10 +43,15 @@ Mappings:
 
   EnvironmentVariables: # This is all the environment specific environment variables that don't belong in globals.
     dev:
+      ACCOUNTID: "060113405249"
```

Comment:
You can remove these and just use `${AWS::AccountId} `directly in your statements its a value AWS sets by default